torch/_dynamo/variables/streams.py

```diff
@@ -60,6 +60,9 @@ def _(
     pass
 
 
+has_side_effect(torch.ops.streams.fork.default)
```

Comment:
I am a tad unhappy with this (adding function call in the operator registration just seems a bit... hacky)

Can we not use `has_side_effect` as a decorator above?

Reply:
no unfortunately because those defs aboe are actually not the exact op that will be seen in the graph. the op overload packet is created w/ mystical custom op magic from the registrations